runtests.sh

```diff
@@ -20,16 +21,13 @@ doUnitTests=true
 
 doCodeFormatFix=false
 doBlackFormat=false
+doFlake8Format=false
 doPytypeFormat=false
 doMypyFormat=false
-doFlake8Format=false
-clean_cache_files=false
 
-NUM_PARALLEL=1
-
-# testing command to run
-cmd="python3"
+doCleanup=false
```

Comment:
This initial value should be at line 27.

Reply:
Fixed